src/test/java/com/mabl/integration/jenkins/MablRestApiClientTest.java

```diff
@@ -1,18 +1,17 @@
 package com.mabl.integration.jenkins;
 
+import com.github.tomakehurst.wiremock.stubbing.Scenario;
 import com.mabl.integration.jenkins.domain.CreateDeploymentResult;
 import com.mabl.integration.jenkins.domain.ExecutionResult;
 import org.junit.Test;
 
 import java.io.IOException;
 
-import static com.github.tomakehurst.wiremock.client.WireMock.notFound;
-import static com.github.tomakehurst.wiremock.client.WireMock.ok;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
```

Comment:
My apologies for the unstated rules, but we try not to use wildcard imports to preserve history and readability. 

`import static com.github.tomakehurst.wiremock.client.WireMock.*;`
`import static org.junit.Assert.*;`